sdk/hatchery/starter.go

```diff
@@ -48,6 +48,12 @@ func PanicDump(h Interface) func(s string) (io.WriteCloser, error) {
 	}
 }
 
+var starters map[int]starter
```

Comment:
where is it used ? I miss a handler, a status or some metrics using those statuses